apiserver/facades/client/client/status.go

```diff
@@ -956,6 +978,10 @@ func (context *statusContext) processApplication(application *state.Application)
 		processedStatus.WorkloadVersion = versions[0].Message
 	}
 
+	if endpointBindingSpace, ok := context.allAppsUnitsCharmBindings.endpointBindings[application.Name()]; ok {
```

Comment:
you don't need the if statement
processStatus.EndpointBindings = allAppsUnitsCharmBindings.endpointBindings[application.Name()]